AndroidXCI/lib/src/test/kotlin/dev/androidx/ci/testRunner/TestMatrixStoreTest.kt

```diff
@@ -220,6 +221,45 @@ internal class TestMatrixStoreTest {
         assertThat(reUploadedAfterDeletion.testMatrixId).isNotEqualTo(testMatrix.testMatrixId)
     }
 
+    @Test
+    fun dontCacheInvalid() = runBlocking<Unit> {
```

Comment:
nit: this test name is a little confusing

Reply:
renamed to `dontReuseTestMatricesWithInfraFailures`